SAMtags.tex

```diff
@@ -208,6 +212,16 @@ \subsection{Additional Template and Mapping data}
 \subsection{Metadata}
 
 \begin{description}
+
+\item[FT:Z:\tagvalue{text}] Filter field, free-text indicating the reason why this record has been modified or filtered
+There are two ways in which reads can be filtered: their 0x200 bit can be set (not passing filters) or their 0x4 bit can be set (segment unmapped). 
```

Comment:
Eeek!  This is going to cause a lot of confusion.  Setting "segment unmapped" is _not_ "filtering" a read.  It's marking it as unmapped.  We should not conflate the two.